app/sidekiq/decision_review/submit_upload.rb

```diff
@@ -1,73 +1,100 @@
 # frozen_string_literal: true
 
 require 'decision_review_v1/service'
+require 'decision_review_v1/utilities/logging_utils'
 
 module DecisionReview
   class SubmitUpload
     include Sidekiq::Job
+    include DecisionReviewV1::Appeals::LoggingUtils
     STATSD_KEY_PREFIX = 'worker.decision_review.submit_upload'
 
     sidekiq_options retry: 5
 
+    sidekiq_retries_exhausted do |_msg, _ex|
+      StatsD.increment("#{STATSD_KEY_PREFIX}.permanent_error")
+    end
```

Comment:
Builds off of Annie's work, and Kyle's comment, in https://github.com/department-of-veterans-affairs/va.gov-team/issues/66046. Our sidekiq error alerts will hinge off this new metric, and the other error logs can be used to aid subsequent investigations.